CRM/Upgrade/Incremental/php/FiveSixtyTwo.php

```diff
@@ -39,6 +39,13 @@ public function setPreUpgradeMessage(&$preUpgradeMessage, $rev, $currentVer = NU
         }
       }
     }
+
+    $activityCount = CRM_Core_DAO::singleValueQuery('SELECT MAX(id) FROM civicrm_activity');
```

Comment:
Thanks for doing this. We only want to display this rerun though if they actually have old revisions in the db. So suggest:
```suggestion
    $activityCount = CRM_Core_DAO::singleValueQuery('SELECT MAX(id) FROM civicrm_activity WHERE is_current_revision = 0');
```